pkg/storaged/client.js

```diff
@@ -411,13 +410,9 @@
         }
 
         function enable_clevis_features() {
-            if (window.localStorage["clevis-feature"] != "on")
-                return cockpit.resolve();
-
-            return cockpit.spawn([ "which", "clevis" ], { err: "ignore" }).then(
+            return cockpit.spawn([ "which", "clevis-bind-luks" ], { err: "ignore" }).then(
```

Comment:
> I have clevis and a dozen clevis-* commands on Fedora 28, but not this one. Expected? 

It comes in the `clevis-luks` package.  They have renamed it later to `clevis-luks-bind`, but kept the old name.

Reply:
Added to spec.

Reply:
I have used "Recommends" instead of "Suggests", but only on rhel-x and fedora.  Does that mean that clevis-luks will be installed by default when cockpit-storaged is installed?  That would be preferrable, I think, so that people actually see the new feature that we have worked so hard on. :)